src/Microsoft.Health.Fhir.ValueSets/AuditEventSubType.cs

```diff
@@ -35,5 +35,7 @@ public static class AuditEventSubType
         public const string SearchSystem = "search-system";
 
         public const string Capabilities = "capabilities";
+
+        public const string ReadCompartmentByResourceType = "read-compartment";
```

Comment:
this is not a valid value i don't think: https://www.hl7.org/fhir/valueset-audit-event-sub-type.html. I think compartment falls under "search".